scripts/mkpdf.js

```diff
@@ -1,11 +1,20 @@
 (async function() {
   const puppeteer = require('puppeteer');
   const path = require('path');
-  const getContent = require('./mk');
+  const getContent = require('./mk-template');
 
-  const CATEGORIES_PDF_FILE_NAME = 'categories/categories.pdf';
+  const CATEGORIES_PDF_FILE_NAME = path.resolve(
+    __dirname,
+    '..',
+    'categories/categories.pdf',
+  );
   const TEMPLATE_PATH = path.resolve(__dirname, 'templates/pdf-template.html');
 
+  const DIRECTORY_NAME = 'categories';
+  if (!fs.existsSync(DIRECTORY_NAME)) {
```

Comment:
Seems you forgot to require this dep above. Everyhting works find when I added that. I can;t push to your PR so can you fixed it?